src/test/regress/expected/create_null_dist_key.out

```diff
@@ -756,6 +767,191 @@ SELECT create_distributed_table('partition_with_null_key', NULL, distribution_ty
+    FOR VALUES IN (1);
+CREATE TABLE "NULL_!_dist_key"."partition2_nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789"
+    PARTITION OF "NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789"
+    FOR VALUES IN (2);
+CREATE TABLE "NULL_!_dist_key"."partition100_nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789"
+    PARTITION OF "NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789"
+    FOR VALUES IN (100);
+-- create some objects before create_distributed_table
+CREATE INDEX "my!Index1New" ON "NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789"(id) WITH ( fillfactor = 80 ) WHERE  id > 10;
+CREATE UNIQUE INDEX uniqueIndexNew ON "NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789" (id);
+-- ingest some data before create_distributed_table
+set client_min_messages to ERROR;
+INSERT INTO "NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789" VALUES (1, 1, row_to_json(row(1,1), true)),
+                                     (2, 1, row_to_json(row(2,2), 'false'));
+reset client_min_messages;
+-- create a replica identity before create_distributed_table
+ALTER TABLE "NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789" REPLICA IDENTITY USING INDEX uniqueIndexNew;
+NOTICE:  identifier "nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789" will be truncated to "nullKeyTable.1!?!9012345678901234567890123456789012345678901234"
+-- test triggers
+SET client_min_messages TO ERROR;
+CREATE FUNCTION insert_id_100() RETURNS trigger AS $insert_100$
+BEGIN
+    INSERT INTO "NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789" VALUES (100, 1, row_to_json(row(1,1), true));
+    RETURN NEW;
+END;
+$insert_100$ LANGUAGE plpgsql;
+CREATE TRIGGER insert_100_trigger
+    AFTER UPDATE ON "NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789"
+    FOR EACH STATEMENT EXECUTE FUNCTION insert_id_100();
+-- error out because of the trigger
+SELECT create_distributed_table('"NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789"', null);
+ERROR:  cannot distribute relation "nullKeyTable.1!?!9012345678901234567890123456789012345678901234" because it has triggers
+HINT:  Consider dropping all the triggers on "nullKeyTable.1!?!9012345678901234567890123456789012345678901234" and retry.
+SET citus.enable_unsafe_triggers TO ON;
+RESET client_min_messages;
+-- this shouldn't give any syntax errors
+SELECT create_distributed_table('"NULL_!_dist_key"."nullKeyTable.1!?!9012345678901234567890123456789012345678901234567890123456789"', null);
```

Comment:
Asking this because I'm seeing the following in the PR description:
>* test triggers

If we're planning to add tests for that in this PR, then we should test and see that such a command is supported when (1) citus.enable_unsafe_trigger is ON (2) but not otherwise.

Reply:
done